README.md

```diff
@@ -1,3 +1,25 @@
 # kecho
 
 This is a lightweight echo server implementation in Linux kernel mode.
+
+## Usage
+
+Run make to build kecho and the benchmarking tool:
+```
+$ make
+```
+
+After the module is loaded, you can use `telnet` to interact with the module:
+```
+$ telnet 127.0.0.1 {port, it's 12345 by default}
```

Comment:
Let's simplify the command line:
```
$ telnet 127.0.0.1 12345
```

You should mention how to change port number as well.

Reply:
I just found module parameter doesn't work due to it's unused. Maybe open another PR to fix this?

In `fastecho_module.c`:
```c
static ushort port = DEFAULT_PORT;
static ushort backlog = DEFAULT_BACKLOG;
module_param(port, ushort, S_IRUGO);
module_param(backlog, ushort, S_IRUGO);
```

```c
addr.sin_port = htons(DEFAULT_PORT);

error = kernel_listen(sock, DEFAULT_BACKLOG);
```

Reply:
Yes, send another pull request for fixing port number assignment.